homeassistant/components/hunterdouglas_powerview/config_flow.py

```diff
@@ -131,15 +151,19 @@ async def async_step_discovery_confirm(self):
         self.powerview_config = {
             CONF_HOST: self.discovered_ip,
             CONF_NAME: self.discovered_name,
+            CONF_API_VERSION: info[CONF_API_VERSION],
         }
         return await self.async_step_link()
 
-    async def async_step_link(self, user_input=None):
+    async def async_step_link(self, user_input=None) -> FlowResult:
```

Comment:
Missing typing on user_input.

I suggest merging all the typing fixes to `dev` first in another PR, which will make this smaller